pkg/master/master.go

```diff
@@ -549,6 +549,46 @@ func (m *Master) InstallSwaggerAPI() {
 	swagger.RegisterSwaggerService(swaggerConfig, m.handlerContainer)
 }
 
+// Picks the best available address for an internal health check
+// TODO: Create utility function, combining this an GetHostIP in kubelet.go?
+func pickAddress(node *api.Node) string {
```

Comment:
I think this code should be functionally identical to that code, but without the temporary map.

GetHostIP is much clearer, so I can do that.  It sounds like maybe we are going to remove this check entirely though, depending on when #7092 lands.  (Some initial discussion in #7115)